analysis/cQED_dataAnalysis/splitQPTdata.m

```diff
@@ -0,0 +1,31 @@
+function [datamat, varmat] = splitQPTdata(data, numPulses, numCals)
+%convert a single-expt. process tomo into a matrix compatible with
+%analyzeProcessTomo
+
+if ~iscell(data)
+   nqubits = 1;
+   datamat = cell(1,1);
+   varmat = cell(1,1);
+   datain = data;
+   data = {};
+   data{1} = datain;
+else
+   nqubits = 2;
+   datamat = cell(3,1);
+   varmat = cell(3,1);
+end
+    
+for kk=1:size(data,2)
+    datamat{kk} = zeros((numPulses^nqubits+numCals),numPulses^nqubits);
```

Comment:
@dieris `numPulses^numqubits` is used enough to warrant being hoisted out and assigned to a temporary